tdrs-backend/tdpservice/email/email.py

```diff
@@ -0,0 +1,92 @@
+"""
+Send emails.
+
+This module currently uses the EmailMultiAlternatives class from Django to send emails with HTML content.
+A optional plain text message can be included as well. Emails should be sent using mail.delay()
+
+The mail function is the main entry point for sending emails.
+It takes in a EmailType enum(that has a template path as the value), a list of recipients,
+and a dictionary of context variables to be used in the email template. The context vlaues can be found in the
+templates directory.
+
+There are optimizations that can be made to this module for sending mass emails.
+Django's send_mass_mail function can be used to send multiple emails at once.
+https://docs.djangoproject.com/en/4.1/topics/email/#send-mass-mail
+"""
+
+from celery import shared_task
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+from django.core.mail import EmailMultiAlternatives
+from django.conf import settings
+from django.template.loader import get_template
+from django.template import TemplateDoesNotExist
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def mail(email_path: str, recipient_email: str, email_context: dict = None) -> None:
```

Comment:
```suggestion
def mail(email_path, recipient_email, email_context=None):
```
This doesn't feel Pythonic -- why are we declaring types everywhere? Would the above still work fine? I see this a recurring theme throughout these functions.

Reply:
In a project this size I am fairly certain that typing and MLA docstrings would increase our development speed. While it takes time to add type hints and docs, once they are there it makes understanding code so much easier because you don't have to track down the source of an object/funtion to know exactly what it does.

Here is an example that uses more complete doc strings and type hints with the mail function I made.
![Screen Shot 2022-09-22 at 8 54 57 AM](https://user-images.githubusercontent.com/7119690/191798112-cc6fc195-43c4-4647-8518-baf82f60fbe5.png)
You can see on the right that when a future developer needs to use this mail function they will be able to see exactly what parameters need to be added.

Here is a good article on the pros and cons of typing.
https://realpython.com/lessons/pros-and-cons-type-hints/

I think that public functions, at least, should be typed so as to give future developers more information when using that function

Reply:
agree with @n0remac on this one! the type hints and docstrings are super helpful to see in the IDE as we're developing with this stuff. i don't think it's worth removing since it's already added.

Reply:
It's not my preference but ok.  Can we please at least remove the function return typing? (e.g. `) -> None:`)